src/vm/exceptionhandling.cpp

```diff
@@ -4644,7 +4676,11 @@ VOID DECLSPEC_NORETURN UnwindManagedExceptionPass1(PAL_SEHException& ex)
             UNREACHABLE();
         }
 
+#ifdef _ARM_
+    } while (IsSpInStackLimits(frameContext.Pc, stackLowAddress, stackHighAddress));
+#else
     } while (IsSpInStackLimits(frameContext.Rsp, stackLowAddress, stackHighAddress));
+#endif
 
     _ASSERTE(!"UnwindManagedExceptionPass1: Failed to find a handler. Reached the end of the stack");
     EEPOLICY_HANDLE_FATAL_ERROR(COR_E_EXECUTIONENGINE);
```

Comment:
General comment.  Wondering if we should consider unifying Pc/Rsp/Eip/etc into a Pc accessor, same for $sp.  Thoughts?  Would clean this up going forward.